internal/utils/eszip/build.ts

```diff
@@ -8,27 +8,42 @@ import { load } from "https://deno.land/x/eszip@v0.30.0/loader.ts";
 const virtualBasePath = "file:///src/";
 
 async function buildAndWrite(p: string) {
+  const funcDirPath = path.dirname(p);
   const entrypoint = new URL("index.ts", virtualBasePath).href;
+
   const eszip = await build([entrypoint], async (specifier: string) => {
     const url = new URL(specifier);
     if (url.protocol === 'file:') {
       console.error(specifier)
       // if the path is `file:///*`, treat it as a path from parent directory
-      let actualPath = specifier.replace('file:///', `./${path.dirname(p)}/../`);
+      let actualPath = specifier.replace('file:///', `./${funcDirPath}/../`);
       // if the path is `file:///src/*`, treat it as a relative path from current dir
       if (specifier.startsWith(virtualBasePath)) {
-        actualPath = specifier.replace(virtualBasePath, `./${path.dirname(p)}/`);
+        actualPath = specifier.replace(virtualBasePath, `./${funcDirPath}/`);
       }
-      const content = await Deno.readTextFile((actualPath));
-      return {
-        kind: "module",
-        specifier,
-        content
+      try {
+        const content = await Deno.readTextFile((actualPath));
```

Comment:
Is the inner bracket redundant?
```suggestion
        const content = await Deno.readTextFile(actualPath);
```